### ISSUE TITLE
README is misleading concerning sample rates

### ISSUE THREAD

**Issue by Evertras** [github/brubeck#40]
Hey there, Brubeck is pretty awesome!  But for a while now we've been under the assumption that it didn't do anything with sample rates due to this line in the readme:

```Client-sent sampling rates are ignored.```

But after digging in the code to check something else, I noticed sample rates are supported when I send something like `test:1|c|@0.1` and get a value of 10 on flush.  Is this line in the README outdated or does it mean something else?

**Comment by vmg** [comment]
You're right! We've actually implemented that feature but forgot to update the README. :sweat:

I'll open a PR tomorrow morning. :)
